components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -523,6 +523,15 @@ GURL MaybeAddInfuraProjectId(const GURL& url) {
   return url;
 }
 
+std::string GetBrave0xAPIKey() {
+  std::string api_key(BUILDFLAG(BRAVE_0X_API_KEY));
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
```

Comment:
can we do this as a command line arg instead of env so it's more explicit and won't be accidentally overridden? I would discourage using env in general in favor of command line options

Reply:
Fixed, thanks!